x-pack/legacy/plugins/monitoring/server/lib/metrics/logstash/metrics.js

```diff
@@ -77,6 +77,15 @@ export const metrics = {
       defaultMessage: 'Number of events emitted per second by all Logstash nodes at the outputs stage.'
     })
   }),
+  logstash_cluster_queue_depth: new LogstashEventsRateClusterMetric({
+    field: 'logstash_stats.queue.events_count',
+    label: i18n.translate('xpack.monitoring.metrics.logstash.queueDepthLabel', {
+      defaultMessage: 'Events Enqueued Rate'
+    }),
+    description: i18n.translate('xpack.monitoring.metrics.logstash.queueDepthDescription', {
+      defaultMessage: 'Events enqueued per second'
```

Comment:
This is typically wordier with more details about what this means, like the `events.in` description says:
> Number of events received per second by all Logstash nodes at the inputs stage.

Maybe something along those lines for this too?

`Number of enqueued events per second by all Logstash nodes`

(I'm not sure if `at the inputs stage` is an appropriate thing to say here or not)